src/inverseRobot.js

```diff
@@ -8,6 +8,17 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newObject = {};
+  const array = Object.entries(robot);
+
+  for (let i = 0; i < array.length; i++) {
```

Comment:
let try to use for in loop to go through object

lear more about `for in`  - [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in)